build.gradle

```diff
@@ -15,6 +17,8 @@ repositories {
 dependencies {
   implementation 'com.google.code.gson:gson:2.8.6'
   implementation 'com.google.guava:guava:28.2-jre'
+  implementation ProjectExtensionsKt.gradleKotlinDsl(project)
```

Comment:
why is this needed?

Reply:
applying `kotlin-dsl` on a _Groovy_ script leads to other issues, here I just need some dependencies provided by the `kotlin-dsl` jar, this approach was suggested by @eskatos

Reply:
SGTM